### ISSUE TITLE
[Bug]: F-Droid offers version 0.119 beta to install but fails on TX8 armv8l... but the armv7 apk does install sideloaded

### ISSUE THREAD

**Issue by rodoviario** [termux/termux-app#4032]
### Problem description

I was using the unsupported 0.118 debug version for more than a year in an old and (I know, usupported) TX8 Android 6.0.1 armv8l device.
Today F-Droid offered me the new 0.119 beta (with restored partial support) and I already was aware of the existence of termux-app_v0.119.0-beta.1+apt-android-5-github-debug_armeabi-v7a.apk in the repo

For installing, F-Droid told me to first uninstall the old one because of mismatched signatures.

* I uninstalled it
* clicked install
* waited for the 102mb download
* answered a last question from the device's os about installing and
* failed with an F-Droid opaque error (sorry, I went on and succeeded sideloading, so didn't take note of the error)

The important fact is that the beta version is listed as compatible with my device (the 0.118 and 0.118.1 are not)
And the termux-app_v0.119.0-beta.1+apt-android-5-github-debug_armeabi-v7a.apk version installs and works afterwards. The v8a doesn't FYI

May be there is not a bug per-se on the app, but surely there is something in the F-Droid store version/manifest that states compatibility and fails 🤷🏻‍♂️

### Steps to reproduce the behavior.

* Use a TX8 armv8l Android 6.0.1 device with latest F-Droid and search for termux

* Either try to update (so be told to uninstall first and then install) or if the device doesn't have Termux installed, proced to click install directly
* Wait for the download
* Answer yes the systems prompt asking if you want to install Termux
* There should appear the error and you go back to the App page with the enabled Install button

### What is the expected behavior?

Not sure, but I guess that one of the 2 following:

* The app should not be listed as compatible, and the button Install should be disabled or unavailable in F-Droid... but considering that one of the apks in github does work... so perhaps the following second option is best:
* The app should succeed installing, perhaps defaulting to the v7a version of the android-5 apk

### System information

* Termux application version: v0.119.0-beta.1
* Android OS version: 6.0.1
* Device model: Amlogic TX8 armv8l

termux-info:
```
$ termux-info
Packages CPU architecture:
arm
Subscribed repositories:
# sources.list
deb https://packages-cf.termux.dev/apt/termux-main-21 stable main
# game-repo (sources.list.d/game.list)
deb https://termux.dev/game-packages-21-bin games stable
# science-repo (sources.list.d/science.list)
deb https://termux.dev/science-packages-21-bin science stable
# root-repo (sources.list.d/root.list)
deb https://termux.dev/termux-root-packages-21-bin root stable
Updatable packages:
All packages up to date
Android version:
6.0.1
Kernel build information:
Linux localhost 3.14.29 #2 SMP PREEMPT Thu Oct 27 11:09:01 CST 2016 armv8l Android
Device manufacturer:
Amlogic
Device model:
TX8

```

**Comment by rodoviario** [comment]
Indeed. Uninstaled termux:boot (and termux)... F-Droid version worked afterwards.
Thanks

**Comment by agnostic-apollo** [comment]
Do not install from F-Droid for android `5`, it supplies the `apt-android-7` variant. I will bump `minSdkVersion` to android 7 in `beta.2` so it's not shown for android `5` on F-Droid and get `beta.1` disabled. Will also set `maxSdkVersion` to android `6` for `apt-android-5` so that it's not installable on Android `7` from github. Forgot to do all that due to rushed release.

**Comment by agnostic-apollo** [comment]
`apt-android-5` variant will only be supplied on GitHub.

**Comment by rodoviario** [comment]
Sorry to bother you with with my lack of knowledge... Should I uninstall the F-Droid version and reinstall the github apk in order for it to be able to bootstrap and actually enter the termux environment?

It was kinda working well before witn the 0.118 github debug version
And the 0.119  bet github debug that I had just installed worked too

**Comment by agnostic-apollo** [comment]
Yes, uninstall and use `apt-android-5` variants from github only for Android 5/6.
